selfservice/flow/settings/handler_test.go

```diff
@@ -228,7 +260,7 @@ func TestHandler(t *testing.T) {
 			returnTo := "https://www.ory.sh"
 			conf.MustSet(ctx, config.ViperKeyURLsAllowedReturnToDomains, []string{returnTo})
 
-			client := testhelpers.NewHTTPClientWithArbitrarySessionToken(t, reg)
+			client := testhelpers.NewDefaultClientWithSessionCookie(t, reg)
```

Comment:
Should be using `NewDefaultClientWithSessionToken`

I missed a lot of these 😬

Reply:
I think this change from token to cookie client is okay, since the we're testing browser flows.